dts/msm8916/msm8916-samsung-r00.dts

```diff
@@ -15,4 +15,16 @@
 		compatible = "samsung,gt58lte-aus", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-bootloader = "T355Y*";
 	};
+
+	coreprimelte {
+		// In the downstream dts it is called Samsung ROSSA PROJECT Rev00
+		model = "Samsung Galaxy Core Prime LTE (SM-G360F)";
+		compatible = "samsung,coreprimelte", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "G360F*";
```

Comment:
You probably want to override this to be like `r01`, like here: https://github.com/msm8916-mainline/lk2nd/blob/5009deaa987e1731f61b92577aa1aeb7d7ee8473/dts/msm8916/msm8916-samsung-r02.dts#L21-L22

Assuming that the Samsung bootloader selects r00 for lk2nd on your device but Rev03 on TWRP.